### ISSUE TITLE
Skip to main link adds a focus ring to the H1

### ISSUE THREAD

**Issue by Cooperwd01** [nhsuk/nhsuk-frontend#864]
We've had a few services in for DAC audits and it's coming back with failures due to the focus ring that's applied around the H1 when a user hits the 'skip to main content' link. DAC auditors are reporting that they only expect focus on interactive elements and find the focus on the h1 to be confusing

It looks like the focus style is added with JS, would it be an easy fix to remove it? Or should the skip to main link just hit main rather than the H1? 

![Screenshot 2023-03-24 at 09 48 10](https://user-images.githubusercontent.com/27928828/227488474-65c3f64e-83ae-4a0d-9d60-d27936d6f792.png)

**Comment by Cooperwd01** [comment]
We recently had some feedback from screen reader users that after hitting the skip link, they'd expect all the content in main to be announced unless they stopped it. Currently, as the focus jumps to the H1, the H1 is announced and then it stops, none of the other the content is read unless a user manually moves to the next part of the content.

**Comment by Cooperwd01** [comment]
@mcheung-nhs That's interesting and makes sense, thanks for the reply. The feedback we received from a few NVDA users was that they would expect the content to continue reading and that they found it was confusing to land directly on the H1. 

If you use NVDA or JAWS on a GOV.UK service and skip to main does the content continue to announce for you? I just had a look and another difference with the GOV.UK skip link is when you click 'skip to main' a tabindex="-1" is added to the main tag. 

Thanks for the help!
